documentation/modules/exploit/winrm/winrm_script_exec.md

```diff
@@ -0,0 +1,67 @@
+## Vulnerable Application
+
+Windows Remote Management (WinRM) is a feature of Windows Vista that allows administrators to remotely run management scripts. It handles remote connections by means of the WS-Management Protocol, which is based on SOAP (Simple Object Access Protocol).   This module uses valid credentials to login to the WinRM service and execute a payload. It has two available methods for payload delivery: Powershell 2.0 and VBS CmdStager. This module will check if Poweshell 2.0 is available, and if so then it will use that method. Otherwise it falls back to the VBS CmdStager which is less stealthy.
```

Comment:
I generally like it when the markdown has a step by step for configuring (if not default), the doc should be self contained as much as possible.
Still not sure if that would be needed, however see https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/exploit/linux/http/centreon_useralias_exec.md#creating-a-testing-environment for an example